client/blocks/comments/post-comment.jsx

```diff
@@ -45,6 +46,8 @@ class PostComment extends Component {
 		maxDepth: PropTypes.number,
 		showNestingReplyArrow: PropTypes.bool,
 		displayType: PropTypes.oneOf( values( POST_COMMENT_DISPLAY_TYPES ) ),
+		toShow: PropTypes.object, // if provided, hide comments not specified in this object
```

Comment:
What's the structure of this object?

Reply:
I would be explicit and call it `commentsToShow` I think.

Reply:
Good question! I'll throw in some comments

```
{
  [ commentId ]: displayType
}
```

Reply:
renamed and comments added in:  https://github.com/Automattic/wp-calypso/pull/17608/commits/5dec64945d07e90f37d373fb2742f5eda70f8bda